helm/policies-aws/tests/abs/test_aws_default.py

```diff
@@ -1,3 +1,6 @@
+import sys
+sys.path.append('../../../tests')
```

Comment:
This was about the only way I could really make this work. The root of the repo does not seem to be available in python and therefor it is in the `helm` subfolder now.